libbeat/cmd/instance/metrics.go

```diff
@@ -30,3 +27,11 @@ func (memstatsVar) Visit(m monitoring.Mode, V monitoring.Visitor) {
 		monitoring.ReportInt(V, "gc_next", int64(stats.NextGC))
 	}
 }
+
+func reportInfo(_ monitoring.Mode, V monitoring.Visitor) {
+	V.OnRegistryStart()
+	defer V.OnRegistryFinished()
+
+	uptime := int64(time.Since(startTime).Seconds()) * 1000
```

Comment:
why use `Seconds()`, do a floor operation and multiply by 1000? 

shouldn't it be:

```
now := time.Since(startTime)
int64(now.Seconds()) * 1000 + int64(now.Nanoseconds() / 1000000)
```